man/systemd.service.xml

```diff
@@ -113,6 +113,18 @@
     involved with early boot or late system shutdown should disable
     this option.</para>
 
+    <para>Note that instanced service units (i.e. service units with an <literal>@</literal> in their name) are by
+    default assigned a per-template slice unit (see
```

Comment:
The "Note that" is mostly just filling, and could go.

I believe "are assigned [...] by default" is better grammar.